LICENSE

```diff
@@ -293,3 +293,29 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 
 -------------------------------------------------------------------------------
+
+This product includes code from org.apache.hadoop.
```

Comment:
@bvaradar : can you check if this is good wrt license

Reply:
Looks good otherwise